### DISCUSSION TITLE
My SSD wont plot

### DISCUSSION THREAD

**Discussion by Pkinch128** [Chia-Network/chia-blockchain#8083] [Support]
My computer has a super fast SSD but i cant figure out this permissions error that stops it from plotting at 0 usually or it will get all the way to 100 and not finish. I have gone into my advanced settings and gave permission but it still wouldnt plot. What am i missing? im not crazy tech savy but i feel like this is an easy fix is there anyone who can help me i have a Dell Optiflex 790

**Comment by BrandtH22** [comment]
@Pkinch128 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - if this issue persists please provide the permissions error that you are seeing and any relevant log outputs for us to provide more accurate support.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
